packages/importer/src/importer.ts

```diff
@@ -148,6 +151,20 @@ export class OneSchemaImporterClass extends EventEmitter {
     }
   }
 
+  _initWithRetry(message: any, count = 1) {
+    if (this._hasLaunched) {
+      return
+    }
+
+    if (count > MAX_LAUNCH_RETRY) {
```

Comment:
- I could just retry forever since it doesn't really matter / shouldn't hurt anything ?
- I could also check for devMode and log that we are retrying if you are in dev mode

- I could make this function a more general purpose util (like retryFn(fn))